src/js/controller/model.js

```diff
@@ -19,6 +19,8 @@ class Model extends SimpleModel {
     }
 
     setup(config) {
+        config = config || {};
```

Comment:
currently this is in place because we have failing unit tests that try to setup a model without a config (previously it wasn't breaking behavior). the alternative is to early exit in `_normalizeConfig`, this seemed like a better catch-all